apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/cohorts/index.vue

```diff
@@ -6,7 +6,8 @@ const router = useRouter();
 const config = useRuntimeConfig();
 const pageSize = 10;
 
-useHead({ title: "Cohorts" });
+const catalogue = route.params.catalogue === "all" ? "" : route.params.catalogue + " ";
```

Comment:
catalogue (string) feels like a confusing variable name here , what is its purpose in the catalogue list page , please use a descriptive name. it it only use to generate a page title ? in that case us local context ( inline ) or named function.

Reply:
indeed, in another screen I called it the 'titleString'